Tests/SentryTests/Integrations/Performance/IO/DataSentryTracingIntegrationTests.swift

```diff
@@ -0,0 +1,236 @@
+@testable import Sentry
+import SentryTestUtils
+import XCTest
+
+class DataSentryTracingIntegrationTests: XCTestCase {
```

Comment:
And same as https://github.com/getsentry/sentry-cocoa/pull/4863#discussion_r1965738399.

Reply:
added cases without SDK and with closed SDK